### DISCUSSION TITLE
Xournal++ 1.0.20-hotfix release: strange file size

### DISCUSSION THREAD

**Discussion by franglais125** [xournalpp/xournalpp#2602] [Q&A] [answered]
The latest 1.0.20-hotifx release has a strange size compared to the latest several releases: 1.3MB as opposed to ~10MB in size for other deb/rpm installation files.

Upon opening the resulting "hotifx" deb, the main binary itself is 90% smaller than it used to be (~2.4MB vs ~27MB).

Per se, this is not an issue, but it would be nice to understand where this difference comes from. Upon compiling locally, I get a deb of ~9MB in size on my Debian bullseye.

Does this require more investigation to ensure that the latest release is not corrupted in some way? Or have we found a very nice filesize optimization/compression somewhere?

**Comment by rolandlo** [comment, marked as answer]
We have [stripped the binaries](https://en.wikipedia.org/wiki/Strip_(Unix)) (from Debug information), since [lintian](https://en.wikipedia.org/wiki/Lintian) asked for it. That shouldn't affect how Xournal++ works. It could only be slightly faster. It may affect the way we can help when there is a problem, though. We will find out.
